packages/e2e/alonzo-private-network/addresses/pool-owner1.addr

```diff
@@ -0,0 +1 @@
+addr_test1qznfezldqf6szyehqter92gaj4sfrdteh055y4pqc03vf7zneppazsprmxajk7r7qd7fjuu8z9vz9c3gmt5378mlhs0qct5w85
```

Comment:
I'm wondering if it would be better to generate all the keys, certs and adresses with a script instead of having them in the repo (not necessarily in scope of this PR). Thoughts @rhyslbw @AngelCastilloB ?

Reply:
This could be done, but there is a benefit on having fixed keys, addresses etc. in the testing environment, this will allow us to make hard assumptions during tests, for example pool ids, genesis keys for the faucet etc. Some of those we probably don't care about though and could be regenerated every run.

Reply:
> this will allow us to make hard assumptions during tests

That's true, but reading them from files would be nice too - then the tests wouldn't need to be updated if any of those values has to change.

Reply:
I will look into this and see how we can do this dynamically then

Reply:
I suggest doing this in a separate ticket/PR